tests/sensors/test_external_task_sensor.py

```diff
@@ -576,6 +610,14 @@ def test_external_task_sensor_waits_for_dag_check_existence(self):
             op.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE, ignore_ti_state=True)
 
 
+def test_external_task_sensor_check_zipped_dag_existence(dag_zip_maker):
+    with dag_zip_maker("test_external_task_sensor_check_existense.py") as dagbag:
+        with create_session() as session:
+            dag = dagbag.dags['test_external_task_sensor_check_existence']
+            op = dag.tasks[0]
+            op._check_for_existence(session)
```

Comment:
Did you miss an `assert` here?

Reply:
_check_for_existence will raise AirflowException when external DAG does not exist.